contrib/root/usr/share/scripts/patroni/post_init.sh

```diff
@@ -2,8 +2,14 @@
 set -Eeu
 
 if [[ (! -z "$APP_USER") &&  (! -z "$APP_PASSWORD") && (! -z "$APP_DATABASE")]]; then
-  echo "Creating user ${APP_USER}"
-  psql "$1" -w -c "create user \"${APP_USER}\" WITH LOGIN ENCRYPTED PASSWORD '${APP_PASSWORD}'"
+
+  if [ "$APP_USER" = "postgres" ]; then
```

Comment:
May be better to consider using the double square brace `[[` notation instead of single square brace `[` notation.
Ref: https://stackoverflow.com/q/3427872